taskboot/git.py

```diff
@@ -130,3 +131,35 @@ def github_release(target, args):
         release.upload_asset(name=asset_name, path=artifact_path, label=asset_name)
 
     logger.info(f"Release available as {release.html_url}")
+
+
+def github_push(args):
+
+    # Load config from file/secret
+    config = Configuration(args)
+    assert config.has_github_auth(), "Missing GitHub authentication"
+
+    # Set remote repository
+    repo_link = "https://{}:{}@github.com/{}.git".format(
+        args.user, config.github["token"], args.repository
+    )
+    subprocess.run(["git", "remote", "set-url", "origin", repo_link])
+
+    # Push on repository
+    try:
+        if args.force_push:
+            subprocess.run(["git", "push", "-f", "origin", args.branch], check=True)
+        else:
+            subprocess.run(["git", "push", "origin", args.branch], check=True)
```

Comment:
Nit: the `subprocess.run(..., check=True)` is redundant, just define a different command variable in each branch and do:
```
subprocess.run(command, check=True)
```